### PR TITLE
German version

### PR DESCRIPTION
**Pull request by 9glenda**

### CODE REVIEW

File: README.de.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+![Dwm](https://raw.githubusercontent.com/antoniosarosi/dotfiles/master/.screenshots/dwm.png)
+
+***Language***
+- [🇪🇸 Español](./README.es.md)
+- [Deutsch](README.de.md)
+- 🇺🇸 English
```
Comment (antoniosarosi) [inline README.de.md]: You're missing the link to the english version here, put [English] inside [] and use parenthesis for the link (https://github.com/antoniosarosi/dwm)

File: README.de.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Die *autostart* Datei befindet sich in **~/.dwm/autostart.sh**.
+Teste es mitTest it with **[Xephyr](https://wiki.archlinux.org/index.php/Xephyr)**:
+
+```bash
+Xephyr -br -ac -noreset -screen 1280x720 :1 &
+DISPLAY=:1 dwm
+```
+
+If you want to modify bar icons, open **~/.config/dwmblocks/config.h**
+and change these lines:
```
Comment (antoniosarosi) [inline README.de.md]: Not translated

File: README.de.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+//   Icon    Command                          Update Interval     Update Signal
+    { "  ", "checkupdates | wc -l",                 60,               0 },
+    { "",    "brightness",                           2,                0 },
+    { "",    "volume",                               2,                0 },
+    { "",    "battery",                              60,               0 },
+    { "",    "date '+ %d/%m/%Y   %H:%M%p'",        5,                0 },
+};
+```
+
+Then recompile *dwmblocks* and restart *dwm* using **mod + control + r**.
```
Comment (antoniosarosi) [inline README.de.md]: Not translated

File: README.de.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+```bash
+cd ~/.config/dwm/dwmblocks
+sudo make clean install
+```
+
+Once that's done, you can login. But keep in mind keybindings will not work
+unless you have the same programs that I use and the same configs. You can
+either change keybindings or install the software I use and my config files,
+check out [my dotfiles repo](https://github.com/antoniosarosi/dotfiles#keybindings)
+for instructions.
```
Comment (antoniosarosi) [inline README.de.md]: Not translated

**Review by antoniosarosi — CHANGES_REQUESTED** [review]

File: README.de.md (hunk 4 above)
Comment (9glenda) [reply README.de.md]: oh thats a false version i am sorry

Comment (antoniosarosi) [reply README.de.md]: Push the final version :)

Comment (9glenda) [reply README.de.md]: i have some issues with emacs i could take some time maybe 5 minutes my new version was destroyed by emas

Comment (antoniosarosi) [reply README.de.md]: No problem, don't rush it, take your time.

**Comment by 9glenda** [comment]
The new Version is on my fork of this repo

**Pull request merged**